### DISCUSSION TITLE
why is it not possible to remove padding and margin in the vuetify table??

### DISCUSSION THREAD

**Discussion by zeusfly-spb** [vuetifyjs/vuetify#13261] [General]
why is it not possible to remove padding and margin in the vuetify table??

**Comment by nekosaur** [comment]
You have not given nearly enough information for anyone to be able to help you. It is very much possible to change padding. Assuming you're talking about the rows, see https://codepen.io/nekosaur/pen/RwodGVe

**Comment by LaurenceHo** [comment]
Yes, you can remove padding in the `v-data-table`. Although you may not be able to configure padding via variable.sass, you still can override CSS like:
```
.v-data-table {
  table {
    tr {
      th, td {
        padding: 0 8px !important;
      }
    }
  }
}
```
Hope this give you some ideas.
